### ISSUE TITLE
20m substrate metadata

### ISSUE THREAD

**Issue by jnephin** [dfo-msea/GIS-Hub-Layers#4]
Update the description of environmental layers used in the model (specifically energy layers). Correct source of data and original resolution.

**Comment by cole-fields** [comment]
will do - for future issues, if you add it to the GIS-Hub Spatial Data Layers projects (on the right hand side), this will automatically get moved to the projects page 
https://github.com/dfo-msea/GIS-Hub-Layers/projects/1#card-32117838

I'm going to see we can default set the issues to move to the project

**Comment by cole-fields** [comment]
I've replaced the ocean energy source reference from Foreman to Masson. Also added the original 3x3 km grid resolution. I'm not touching the rest of the metadata for now as the substrate group is modifying the model. Alot of it will have to go (the MGET Arc tool, 'aligning data'(?), the entire 1-step v 2-step methodology)

**Comment by jnephin** [comment]
Still somewhat confusing because metadata is now a mix of two substrate model versions. Will discuss what version of the substrate model this metadata record is describing.

**Comment by cole-fields** [comment]
Agreed - i think it should only describe the updated product as we are not going to host the old models on the hub.

**Comment by jnephin** [comment]
Decided to move to draft mode on GISHub until substrate model is finished.
